app/_data/tables/features/gateway.yml

```diff
@@ -0,0 +1,128 @@
+columns:
+  - name: Open Source
+    cta: Get Started
+    cta_link: /
```

Comment:
This just goes to the docs.konghq.com landing page - is that on purpose?

Reply:
Pending discussion, I forgot to mark as draft. It's likely to go to an install page, but pending decision

Reply:
Merging to use as a base for the new installation index, as this link will point there.